meta-cli/src/utils/mod.rs

```diff
@@ -61,8 +59,8 @@ pub fn plural_suffix(count: usize) -> &'static str {
 
 #[derive(Debug, Clone)]
 pub struct BasicAuth {
```

Comment:
Why not just re-export?
```
pub use common::node::BasicAuth;
```

Reply:
We have an `From<common::node::BasicAuth>` right after instead.